packages/examples/basic/package.json

```diff
@@ -1,10 +1,15 @@
 {
   "private": "true",
   "scripts": {
-    "start": "react-scripts start"
+    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
+    "build": "yarn build && yarn build:app",
+    "build:blade": "cd ../../blade && yarn build",
+    "build:app": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
+    "analyze:only": "source-map-explorer 'build/static/js/*.js' --gzip",
+    "analyze": "yarn build:app && yarn analyze:only"
```

Comment:
kept it for only app because you will want to run `build:blade` only when something in blade is changed. But in most cases, you would just want to build the app for new changes and analyze that.


So ideally you're expected to run `yarn build:blade` first manually and then do `yarn analyze` after every change in example

Reply:
I will run analyze on this example only when I want to check the bundle size of a particular component I've created on blade. Won't I always need to run `yarn build:blade` for that? What would be the use-cases where I want to run analyze only for changes in the basic example and not for changes in blade?

Reply:
So you will run `yarn build:blade` when you have built the component. After that you won't have to run that again right? unless you change something in blade code.

I think irrespective of that it should be fine to add build like you mentioned since I have also added `analyze:only` script as well for the usecase I was talking about.

Reply:
> unless you change something in blade code

While developing the component, won't that be a frequent thing anyway?

The only reason I'm pushing for adding `yarn build` before analyze is so that none of us get stuck or confused on why some of our components are not visible in the analyze because we forgot to run `yarn build:blade`. Especially when you're jumping branches to do PR reviews and want to analyze the newly built component

Reply:
> While developing the component, won't that be a frequent thing anyway?

True. Depends on whether you're checking bundle after developing component or while developing component

> The only reason I'm pushing for adding yarn build before analyze is so that none of us get stuck or confused on why some of our components are not visible in the analyze because we forgot to run yarn build:blade. Especially when you're jumping branches to do PR reviews and want to analyze the newly built component

Yup this makes sense.